### ISSUE TITLE
Failure to compile

### ISSUE THREAD

**Issue by tumbl3w33d** [amqp-rs/tcp-stream#3]
I'm using `lapin` (with feature `rustls-native-certs`) which in turn uses this crate and I'm running into the following build error now:
```
error[E0599]: no method named `decrypt_padded_vec` found for struct `Decryptor` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/p12-0.6.2/src/lib.rs:660:9
    |
660 |     rc2.decrypt_padded_vec::<Pkcs7>(data).ok()
    |         ^^^^^^^^^^^^^^^^^^ method not found in `Decryptor<Rc2>`

error[E0599]: no method named `decrypt_padded_vec` found for struct `Decryptor` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/p12-0.6.2/src/lib.rs:700:10
    |
700 |     tdes.decrypt_padded_vec::<Pkcs7>(data).ok()
    |          ^^^^^^^^^^^^^^^^^^ method not found in `Decryptor<TdesEde3>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `p12` due to 2 previous errors
```

I open this issue here because this seems to be the end of the line:
```
p12 v0.6.2
└── tcp-stream v0.24.3
    └── amq-protocol-tcp v7.0.0
        └── amq-protocol v7.0.0
            └── lapin v2.0.1
                └── myapp
```
Not sure this is the right place but maybe you could give me a hint.

**Comment by Keruspe** [comment]
You can workaround that by pinning p12 to version 0.6.1

Will publish 0.6.3 with the fix holly

**Comment by Keruspe** [comment]
Fixed in p12 0.6.3
